File: LUA/Cap05/c05ex06.lua
-- c05ex06.lua

resp = "S"
repeat
  io.write("Entre um valor numerico: ")
  n = tonumber(io.read())

  r = n * 3

  print("Resultado = " .. r)
  print()

  io.write("Deseja continuar? (S/N): ")
  resp = io.read():sub(1, 1):upper()

  print()
until (resp ~= "S")

print()
io.write("Tecle <Enter> para encerrar... ")
io.read()

File: PYTHON/Cap05/c05ex06.py
# c05ex06.py

resp = "S"
while True:
  n = int(input("Entre um valor numerico: "))

  r = n * 3
  print("Resultado =", r)
  print()

  resp = input("Deseja continuar? (S/N): ").upper()[0]
  print()
  if not (resp.upper() != "S"):
    continue
  else:
    break

input("\nTecle <Enter> para encerrar... ")
